src/mame/machine/eispc_kb.cpp

```diff
@@ -278,7 +279,7 @@ WRITE_LINE_MEMBER(eispc_keyboard_device::rst_line_w)
 	else
 	{
 		// set_input_line suspends with a true argument which causes "Keyboard error" 
-		m_mcu->suspend(SUSPEND_REASON_RESET, false);
+		m_mcu->suspend(SUSPEND_REASON_RESET, false);  // This causes an assert later when DEBUG==1
```

Comment:
It's really not acceptable to have a driver that crashes on an assertion failure in normal use.  It means you're definitely doing something wrong.  Can you please find some other way to deal with the synchronisation issues?

Reply:
Yes, I just need some time, not sure I'll hit the deadline though so I committed this first. Or we can accept keyboard error for now and just press F1 to get past it.